server/server.go

```diff
@@ -218,11 +218,11 @@ func newServer(ctx context.Context, c Config, rotationStrategy rotationStrategy)
 	if err != nil {
 		return nil, err
 	}
-	handle("/.well-known/openid-configuration", discoveryHandler)
+	handleWithCORS("/.well-known/openid-configuration", discoveryHandler)
 
 	// TODO(ericchiang): rate limit certain paths based on IP.
-	handleFunc("/token", s.handleToken)
-	handleFunc("/keys", s.handlePublicKeys)
+	handleWithCORS("/token", s.handleToken)
```

Comment:
Let's skip this for now. At some point I think we'll want to just allow specific flows instead of response_types. E.g.

```
allowedFlows := []string{"hybrid", "code", "implicit"}
```

When we do that, we can enable /token if "hybrid" is specified